src/cat.ts

```diff
@@ -159,8 +206,8 @@ export class Cat {
   private estimateAbilityEAP() {
     let num = 0;
     let nf = 0;
-    this.prior.forEach(([theta, probability]) => {
-      const like = this.likelihood(theta);
+    this._prior.forEach(([theta, probability]) => {
+      const like = Math.exp(this.likelihood(theta)); // Convert back to probability
```

Comment:
I know this fixes the bug but can we leave it for PR #37 so that this PR only addresses the ability prior?
```suggestion
      const like = this.likelihood(theta);
```

Reply:
Or we can just merge #37 and update this branch and then this goes away.